### ISSUE TITLE
support language: XML

### ISSUE THREAD

**Issue by ShreyashKore** [SnipMeDev/Highlights#61]

**Comment by tmaxxdd** [comment]
Yeah, this requires extra work as library only finds "<" ">" as the special marks. Any contribution from you will be great! :) We can generally think about "Support for markup languages"

**Comment by ShreyashKore** [comment]
Yup.. I thought so... can you point me in a direction from where I can start? If the change is relatively simple... I might give it a shot.

**Comment by tmaxxdd** [comment]
Simplest way may be to add new type of locator for that
